### DISCUSSION TITLE
Neue Hoymiles Firmware mit verbesserter AC Limit Logik

### DISCUSSION THREAD

**Discussion by spcqike** [hoylabs/OpenDTU-OnBattery#1366] [General]
Mahlzeit 

so wie es aussieht bekommen die einige HMS eine neue Firmware, die das hoymiles internen limiter überarbeitet. 
damit sollen wohl nicht mehr die MPPT gleichmäßig limitiert werden, sondern der AC Ausgang. 

Ich lass mich mal überraschen, was das bewirkt und wie die neuen Modelle sich damit am Ende tatsächlich verhalten 

https://www.photovoltaikforum.com/thread/236667-power-distribution-logik-for-hms-1600-1800-2000-4t/

ebenfalls wurden dort neue Modelle mit Bluetooth angekündigt, die noch nicht gelistet sind.

**Comment by schlimmchen** [comment]
Jup, das ging auch schon durch den discord und ist eine spannende Geschichte!

**Reply by spcqike** [reply]
Hat schon jemand getestet, wie schnell die Firmware regelt? Ein anderer Hersteller hatte sowas auch schon mal, die YouTube Videos dazu haben mich aber nicht überzeugt. Die Modelle brauchten teilweise 30s zum einregeln, wenn ich mich richtig erinnere.


Und, will man das in die Dokumentation aufnehmen? Immerhin ist die Option der überskalierung damit hinfällig. Aber eben nur wenn man die neue Firmware hat (:

**Reply by spcqike** [reply]
Oh, und wie wirkt sich das am Ende auf Batterie Systeme aus, wo leichte Unterschiede zwischen den DC Leitungen / Längen existieren? Wir wissen ja, dass manche User da ihre Ziel Limits nicht erreichen (wegen der geteilten Masse und dem internen falschen messen der Ströme). 

Ich bin mir grad nicht sicher, ob die neue Firmware da gut ist und zuspielt oder nicht :)

**Comment by stefan123t** [comment]
@spcqike hier gibt es auch schon ein issue https://github.com/tbnobody/OpenDTU/issues/2374 upstream wo erste Erfahrungsberichte und der Request nach Aufnahme der neuen Hardware Part/Teilenummer 124097 angefragt wird. Und hier der Erfahrungsbericht https://github.com/tbnobody/OpenDTU/issues/2362

Bernhard broth-itk hat auch schon das Firmware Update mitgeschnitten, aber mE sollten wir vorher das Grid Profile Update implementieren und sauber hinbekommen.

**Comment by jab42** [comment]
Wie ist es denn hiermit weitergegangen?

**Reply by spcqike** [reply]
die neue Logic (PDL, power-distribution-logic, nicht zu verwechseln mit unserem DPL)  ist in der Firmware 01.01.12 integriert. seit einigen releases wird dies auch so unterstützt

https://github.com/hoylabs/OpenDTU-OnBattery?tab=readme-ov-file#inverter-firmware-compatibility

bei Wechselrichtern mit der FIrmware ist im DPL beispielsweise die Option der "Verschattung ausgleichen" entfernt, da der WR das nun alleine machen sollte.

**Reply by jab42** [reply]
Würdest du denn bei einem HMS1600 das Update empfehlen?

**Reply by spcqike** [reply]
solange es mit der alten Version und dem "Verschattung ausgleichen" problemlos läuft, würde ich es nicht updaten. "never change a running system"

Allerdings hab ich in letzter Zeit kaum noch Probleme von Usern gelesen, die die 01.01.12 haben. Also sollte es wohl hoffentlich laufen wie gedacht.
